Revit_Engine/Revit_Engine.csproj

```
@@ -81,6 +81,11 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Geometry_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Graphics_oM, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\..\ProgramData\BHoM\Assemblies\Graphics_oM.dll</HintPath>
```

```suggestion
      <HintPath>C:\ProgramData\BHoM\Assemblies\Graphics_oM.dll</HintPath>
```
This annoying path slipped through 
😄

---

@pawelbaran 👆

---

Well spotted, thank you @al-fisher! I told myself 'do not forget about changing the reference path'... 😆